### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by adpgithub**

### CODE REVIEW

**Review by arozumenko — CHANGES_REQUESTED** [review]
Why you are commenting code in dialogs - it will generate not very appealing output

**Comment by BobMuenchen** [comment]
I agree that we should almost never have comments in dialogs. That's what
help files are for.

**Comment by adpgithub** [comment]
> Why you are commenting code in dialogs - it will generate not very appealing output

I will confirm with Aaron.

File: mergeDatasets.js [hunk 1]
```diff
@@ -153,7 +157,9 @@ BSkyLoadRefreshDataframe( "{{selected.out | safe}}" )
                     label: localization.en.by,
                     placeholder: "",
                     allow_spaces:true,
-                    extraction: "TextAsIs",
+                    //extraction: "TextAsIs",
```
Comment (arozumenko) [inline mergeDatasets.js]: you better remove then comment - it could result in a lot of commented settings and looks quite unprofessional

**Review by arozumenko — CHANGES_REQUESTED** [review]

File: mergeDatasets.js (hunk 1 above)
Comment (adpgithub) [reply mergeDatasets.js]: Done.

**Pull request merged**